containers/Dash/Users/index.js

```diff
@@ -39,6 +39,10 @@ class Users extends React.Component {
 
   render() {
     const { users } = this.props
+
+    // Sort by username
+    users.sort(function(a,b) {return (a.username > b.username) ? 1 : ((b.username > a.username) ? -1 : 0);} );
+
```

Comment:
This should be after the `if (!users) ...` check below because users might be `null` and not an array.